src/Microsoft.DotNet.Build.Tasks.net45/ExtractResWResourcesFromAssemblies.cs

```diff
@@ -43,6 +44,12 @@ public void CreateReswFiles()
             foreach (ITaskItem assemblySpec in InputAssemblies)
             {
                 string assemblyPath = assemblySpec.ItemSpec;
+
+                if (!ShouldExtractResources($"FxResources.{Path.GetFileNameWithoutExtension(assemblyPath)}.SR.resw", assemblyPath))
+                {
+                    continue; // we skip framework assemblies that resources already exist and don't need to be extracted to avoid reading dll metadata.
+                }
+                
                 try
                 {
                     Assembly assembly = Assembly.LoadFrom(assemblyPath);
```

Comment:
`Assembly.LoadFrom` will throw a `BadImageFormatException` which I'm catching and ignoring and going to the next assembly:

https://github.com/dotnet/buildtools/pull/1630/files#diff-4dba46151dfcc864a9186f4ae3de64c1R84